app/forms/delivery_partner_form.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class DeliveryPartnerForm
+  include ActiveModel::Model
+
+  attr_accessor :name, :lead_providers, :provider_relationships
+
+  def available_lead_providers
+    LeadProvider.select { |lp| lp.cohorts.any? }
```

Comment:
I know what 'lp' is, you know what 'lp' is, but I prefer full words :P 
`lead_provider` please :)